neasqc_wp61/data/data_processing/use_beta_1.py

```diff
@@ -4,8 +4,8 @@
 import time 
 import json
 current_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(current_path + "/../../models/quantum/beta/")
-from QuantumKNearestNeighbours import QuantumKNearestNeighbours as qkn
+sys.path.append(current_path + "/../../models/quantum/beta_1/")
+from beta_1 import QuantumKNearestNeighbours as qkn
```

Comment:
/!\ Here in the set_up_exp PR, the beta folder + beta files are renamed. It's not the case for the new general_quantum_distance PR.